spec/javascripts/components/layout-super-navigation-header-spec.js

```diff
@@ -232,117 +673,289 @@ describe('The super header navigation', function () {
     document.body.removeChild(container)
   })
 
-  describe('the Menu button', function () {
-    var $button
-
+  describe('on small screens', function () {
     beforeEach(function () {
-      $button = document.querySelector('.gem-c-layout-super-navigation-header__menu-button')
-    })
+      // `windowSize` returns `'mobile'` or `'desktop'` depending on the screen
+      // size. As we can't change the size of the browser window, we need to
+      // change the return of this function to force the mobile or desktop
+      // behaviour.
+      thisModule.windowSize = function () {
+        return 'mobile'
```

Comment:
Sneaky

Reply:
To be honest m not sure that this is the best way of faking it, but it works so until it breaks 🤷